src/wp-includes/script-loader.php

```diff
@@ -2845,8 +2862,6 @@ function wp_get_inline_script_tag( $javascript, $attributes = array() ) {
 	 */
 	$attributes = apply_filters( 'wp_inline_script_attributes', $attributes, $javascript );
 
-	$javascript = "\n" . trim( $javascript, "\n\r " ) . "\n";
```

Comment:
Note: This needs to move up above the applying of the `wp_inline_script_attributes` filters so that the final `$javascript` is available for computing a CSP hash.